e2e/e2e-local.sh

```diff
@@ -6,4 +6,4 @@ set -euxo pipefail
 : "${PARALLEL:=100}"
 
 go version
-go test -parallel $PARALLEL -timeout $TIMEOUT ./...
+go test -parallel $PARALLEL -timeout $TIMEOUT -v ./...
```

Comment:
this isn't too verbose preventing us from properly seeing the test results ?

Reply:
I spoke with Artur who said he removed it accidently. Will revert - was trying to get "in progress" messages rather than the scripts just block for a couple of mins as the VMs start up.